shared_libraries/orca_shared/recovery/__init__.py

```diff
@@ -5,6 +5,6 @@
     RequestMethod,
     create_status_for_job,
     get_aws_region,
-    post_entry_to_queue,
+    post_entry_to_fifo_queue,
```

Comment:
Need to add `post_entry_to_standard_queue` to the list.

Reply:
oops added